spec/concepts/user/find_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe TodoList::Find do
```

Comment:
We're inside `user/find_spec`. May I know why we're specifying `TodoList::Find`?